public/data/transports.json

```diff
@@ -11,28 +12,33 @@
     "ET": { "road": null, "sea": 4220, "air": 4067 },
     "FR": { "road": 1550, "sea": 4913, "air": 1554 },
     "GB": { "road": 2179, "sea": 4777, "air": 2183 },
-    "GR": { "road": 280, "sea": 1067, "air": 280 },
-    "HU": { "road": 667, "sea": 806, "air": 667 },
+    "GR": { "road": 454, "sea": 1067, "air": 280 },
+    "HU": { "road": 806, "sea": 806, "air": 667 },
     "IN": { "road": 5823, "sea": 7661, "air": 5830 },
-    "IT": { "road": 627, "sea": 550, "air": 628 },
+    "IT": { "road": 727, "sea": 550, "air": 628 },
+    "KE": { "road": null, "sea": 6961, "air": 4787 },
     "KH": { "road": 8649, "sea": 11995, "air": 8658 },
     "LK": { "road": 7083, "sea": 8127, "air": 7084 },
     "MA": { "road": null, "sea": 2746, "air": 2441 },
     "MM": { "road": 7446, "sea": 10636, "air": 7459 },
     "NL": { "road": 1671, "sea": 5089, "air": 1673 },
+    "NZ": { "road": null, "sea": 20022, "air": 17843 },
+    "PE": { "road": null, "sea": 12940, "air": 11236 },
     "PK": { "road": 4622, "sea": 8483, "air": 4631 },
     "PL": { "road": 1223, "sea": 6662, "air": 1223 },
     "PT": { "road": 2372, "sea": 3088, "air": 2378 },
-    "RO": { "road": 687, "sea": 1668, "air": 687 },
+    "RO": { "road": 827, "sea": 1668, "air": 687 },
     "TN": { "road": null, "sea": 1188, "air": 1243 },
     "TR": { "road": 1295, "sea": 1760, "air": 1298 },
     "TW": { "road": 9180, "sea": 14141, "air": 9196 },
+    "US": { "road": null, "sea": 11289, "air": 9174 },
     "VN": { "road": 8520, "sea": 12819, "air": 8531 }
   },
   "BD": {
     "BE": { "road": 7691, "sea": 14433, "air": 7704 },
+    "BR": { "road": null, "sea": 19020, "air": 16009 },
     "CH": { "road": 7495, "sea": 11409, "air": 7508 },
-    "CN": { "road": 1894, "sea": 9244, "air": 1894 },
+    "CN": { "road": 2936, "sea": 9244, "air": 1894 },
```

Comment:
@paulboosz is it expected you changed distances between Bangladesh and China? This updates makes an end-to-end test fail

![2022-12-20 at 14 35](https://user-images.githubusercontent.com/41547/208680154-dd948ab8-05cb-4f3a-bdc0-a4294654c619.png)

Reply:
Searates API has updated their distances. Will update tests expected values.